assets/js/modules/analytics/datastore/properties.js

```diff
@@ -511,18 +371,34 @@ export const selectors = {
 	 * @param {string} accountID The Analytics Account ID to check for property creation.
 	 * @return {boolean} `true` if fetching a properties, `false` if not.
 	 */
-	isDoingGetProperties( state, accountID ) {
-		const { isFetchingPropertiesProfiles } = state;
+	isDoingGetProperties: createRegistrySelector( ( select ) => ( state, accountID ) => {
+		// Check if dispatch calls right after fetching are still awaiting.
+		if ( state.isAwaitingPropertiesProfilesCompletion ) {
+			return true;
+		}
```

Comment:
I think this flag isn't necessary and may lead to a race condition where two requests could overwrite the value here one way or the other since it isn't keyed by the args.

Instead, I think we could simply return `select( STORE_NAME ).isResolving( 'getProperties', [ accountID ] )`. That might mean that in this case we don't use the `isFetchingGetPropertiesProfiles` selector where we would normally, but I think that's okay. If the resolver is resolving `getProperties` for the given account ID, then we can consider `isDoingGetProperties` true.

Reply:
See my other comment in response to your suggestion, I think that's a good point (and I don't like the flag used here and in `accounts.js` overall), but we should look into it separately.

For this specifically, it makes sense these requests could easily conflict, so I've fixed it by keying by `accountID`.